source/javascripts/services/_appService.js.erb

```diff
@@ -564,6 +566,40 @@
 				return $q.when();
 			};
 
+			appService.getBetaTags = function() {
+				if (appService.betaTags) {
+					return $q.when();
+				}
+
+				if (requestService.mode === "website") {
+					return requestService.getBetaTags()
+						.then(function(betaTags) {
+							appService.betaTags = betaTags;
+						}).catch(() => {
+							appService.betaTags = null;
+						});
+				}
+				appService.betaTags = null;
```

Comment:
I think this can be omitted, so it simply remains `undefined`.

Reply:
removed changes to app service as I'm using the request service directly in Stack.getAll